src/test/java/com/palantir/docker/compose/execution/DockerComposeTest.java

```diff
@@ -124,4 +125,26 @@ public void when_there_is_no_container_found_for_ports_an_i_s_e_is_thrown() thro
         compose.ports("db");
     }
 
+    @Test
+    public void docker_compose_exec_pass_concatenated_arguments_to_executor() throws IOException, InterruptedException {
+        when(executedProcess.getInputStream()).thenReturn(toInputStream("docker-compose version 1.7.0rc1, build 1ad8866"));
+        compose.exec(new DockerComposeExecOption(new String[] {"-d"}), "container_1",
+                     new DockerComposeExecArgument(new String[] {"ls"}));
+        verify(executor).execute("-v");
```

Comment:
This check doesn't belong in this test, I think it's basically covered by the next test.